setup.cfg

```diff
@@ -86,9 +86,8 @@ install_requires =
     blinker
     cached_property~=1.5;python_version<="3.7"
     cattrs~=1.1, !=1.7.*
-    # Required by vendored-in connexion
-    clickclick>=1.2
```

Comment:
@mik-laj @ephraimbuddy -  I am not sure if some of those should remain ?

Reply:
We should remove `clickclick` from aiirflow.'s setup.cfg file. Thiis library is only used by connexion CLI. Is not used by Airflow.
https://github.com/zalando/connexion/blob/2066503c5c9f30c2aaf98ad853ff8e16edd64826/connexion/cli.py#L11